### DISCUSSION TITLE
Which pins correspond to the GPIO pins on the Raspberry Pi Pico?

### DISCUSSION THREAD

**Discussion by henryernice** [Bodmer/TFT_eSPI#2907] [Raspberry Pi Pico]
In the User_Setup.h file you can set the pins like mosi, miso, dc etc. but it doesn't state which pins to use for Raspberry Pi Pico.
And another thing is that i downloaded the library and i don't even know how to modify the user_setup.h file.

**Comment by larrybud2004** [comment]
You need to look in the "user_setups" folder in the library for the header file "Setup60_RP2040_ILI9341.h"

Are you using the Arduino ide? If so, check the "libraries" folder of your arduino projects folder. You'll see the TFT_eSPI folder in there with all the header files.

**Reply by henryernice** [reply]
I have now found the header file and have set the pins to the correct ones on my Raspberry pi pico but i cant get any of the example libraries to load. I just have a blank white screen. I dont really know what i could say to specify my problem but i really hope you can help me. 
Im not too good at this stuff but please bear with me.

And the connection between my computer and pico is good since i can load a standard blinking program onto it.
